libs/ledger-live-common/src/families/cosmos/specs.ts

```diff
@@ -430,17 +431,28 @@ function cosmosLikeMutations(minimalTransactionAmount: BigNumber): MutationSpec<
   ];
 }
 
-const generateGenericCosmosTest = (currencyId: string, config?: Partial<AppSpec<Transaction>>) => {
+const generateGenericCosmosTest = (
+  currencyId: string,
+  isExpertModeRequired: boolean,
```

Comment:
I'm thinking `isExpertModeRequired` should also belong to `AppSpec`. There are already a couple custom config params there, and I believe it's better to keep them all centralized.

Reply:
I am not sure since it's cosmos specific